src/sentry/static/sentry/app/stores/tooltipStore.tsx

```diff
@@ -0,0 +1,41 @@
+import Reflux from 'reflux';
+
+import Tooltip from 'app/components/tooltip';
+
+type TooltipStoreInterface = {
+  addTooltip: (tooltip: Tooltip) => void;
+  removeTooltip: (tooltip: Tooltip) => void;
+  openAllTooltips: () => void;
+  tooltips: Tooltip[];
+};
+
+const tooltipStore: Reflux.StoreDefinition & TooltipStoreInterface = {
```

Comment:
You could keep the same interface and remove the reflux dependency completely since nothing needs to listen to the store.